lib/matplotlib/bezier.py

```diff
@@ -111,7 +111,7 @@ def split_de_casteljau(beta, t):
 
 
 def find_bezier_t_intersecting_with_closedpath(
-        bezier_point_at_t, inside_closedpath, t0=0., t1=1., tolerance=0.01):
+        bezier_point_at_t, inside_closedpath, t0=0., t1=1., tolerence=0.01):
```

Comment:
You cannot simply rename parameters in public functions, because this will break user code. Such a change has to go through a [deprecation process](https://matplotlib.org/devdocs/devel/api_changes.html#deprecate-api). We could to this, but it's a separte topic that should be handled in a dedicated PR.